adapter/src/test/java/com/twilio/kudu/sql/ScenarioIT.java

```diff
@@ -45,6 +46,7 @@
 
   @BeforeClass
   public static void setup() throws SQLException, IOException {
+    KuduPrepareImpl.initializeHints();
```

Comment:
This static function is only called during tests. Does that mean it isn't needed for normal runtime? Could we just move that public static function into this test?

Reply:
This is needed if we want to be able to use `USE_KUDU_NESTED_JOIN` hint while executing a sql statement. However 
since it uses `setAccessible` the call might fail depending on the jvm settings. 
https://stackoverflow.com/questions/3301635/change-private-static-final-field-using-java-reflection

Adding the below to KuduQuery allows us to use the hint from RelBuilder without calling this static function.
```
getCluster().setHintStrategies(KUDU_HINT_STRATEGY_TABLE);
```

 I haven't figured out a clean way to enable using the hint while executing a sql statement. Will move it to the test for now.